python/test/ngraph/test_ops_unary.py

```diff
@@ -20,7 +20,6 @@
 from test.ngraph.util import run_op_numeric_data, run_op_node
 
 
-@pytest.mark.xfail(reason='Results mismatch when passing created Constant node from raw data.')
 @pytest.mark.parametrize('ng_api_fn, numpy_fn, range_start, range_end', [
     (ng.cos, np.cos, -100., 100.),
     (ng.sin, np.sin, -100., 100.),
```

Comment:
I'd remove `test_unary_op_array_err` test since it's solved and update `test_unary_op_array` data ranges for respective functions.